lib/mariaex/messages.ex

```diff
@@ -262,6 +268,16 @@ defmodule Mariaex.Messages do
   defp decode_boolean("1"),    do: true
   defp decode_boolean("0"),    do: false
 
+  defp decode_decimal(data) do
+    data
+    |> String.codepoints
+    |> Enum.reverse
+    |> trim_zeros
+    |> Enum.reverse
+    |> List.to_string
+    |> Decimal.new
```

Comment:
Yeah, that's true.
I will change the code and add more test cases to cover these points.

Thanks again :)